packages/mask/src/plugins/Saving/SNSAdaptor/saving/MarketHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import { makeStyles } from '@masknet/theme'
+import { useI18N } from '../../../../utils'
+
+const useStyles = makeStyles()((theme) => ({
+    wrap: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        background: theme.palette.mode === 'light' ? '#F6F8F8' : '#17191D',
```

Comment:
Avoid using dead code color.

```suggestion
        background: theme.palette.background.paper,
```